apps/site/README.md

```diff
@@ -265,19 +265,40 @@ Retrieves the user object of the currently logged in user.
 Retrieves metadata on the API keys associated with the authenticated user.
 
 - Request Response:
-  - `apiKeys`: metadata on API keys (the key itself is only visible at the point of generation)
+  - `apiKeys`: metadata on all the user's API keys (the keys themselves are only visible at the point of generation)
 
 #### `POST /api/me/generate-api-key` [requires cookie authentication]
 
-Generates a new API key for the authenticated user, and revokes any others.
+Generates a new API key for the authenticated user.
 
 - Request Body:
 
-  - `displayName`: a display name for the API key
+  - `displayName`: a display name for the API key.
 
 - Request Response:
   - `apiKey`: the key itself, a string.
 
+#### `POST /api/me/revoke-api-key` [requires cookie authentication]
+
+Revokes a given API key for the authenticated user.
+
+- Request Body:
+
+  - `publicId`: the public ID portion of the API key to revoke.
+
+- Request Response: Empty
```

Comment:
It's not really an empty response, it's `"{}"`. We could instead make it `true` (i.e. `res.status(200).send(true)`) or `"SUCCESS"` (we do this for logout) or something. I think it's fine to be an empty object too, since basically if the request doesn't error it has succeeded, but we should describe it as such here.

Reply:
Changed to "SUCCESS" in a30e7f5eaf912f1d03ce4f541f4b04b6c2ea34e3 for consistency with logout. I think `true` would be fine too, but both are strongly typed.

I didn't go with `{}` because it's catch-all type and we don't get errors in the `.send` if we provide another value.